wgsl/index.bs

```diff
@@ -2972,7 +2972,7 @@ variable_statement
   | LET (IDENT | variable_ident_decl) EQUAL short_circuit_or_expression
 
 variable_decl
-  : VAR variable_qualifier? variable_ident_decl
+  : VAR variable_qualifier? (IDENT | variable_ident_decl)
```

Comment:
Why is this change needed?
Does it change the grammar?

Reply:
@dneto0 yes, the other version doesn't allow parsed syntax to accept `var anIdent = typeInferrableValue;`, but maybe there is something the script misinterprets about `variable_ident_decl` that should actually allow this idiom?

Reply:
Ok, maybe we forgot to update the grammar when we added type inference for var.  I'll look more closely.

Reply:
Right, I forgot to update the grammar as part of #1756 

Looks good to me, then.

Reply:
@dneto0 thank you for checking!